src/components/LoginPage/Slide4.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import slideImg1 from 'components/LoginPage/images/slide-4_img-1.svg';
+import slideImg2 from 'components/LoginPage/images/slide-4_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 10%;
+  text-align: center;
+  position: relative;
+
+<<<<<<< HEAD
```

Comment:
Please resolve the conflict :)

Reply:
done in commit [e05031d](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/e05031dc46bd06881abd4aca71e972ea7115502f)